phalcon/mvc/model/query.zep

```diff
@@ -2513,6 +2522,93 @@ class Query implements QueryInterface, InjectionAwareInterface
 	}
 
 	/**
+	 * gets the read connection from the model if there no transaction inside the query object
```

Comment:
We you the PHPDoc to generate API documentation and IDE Stubs. Thus could you please provide small examples for the key methods and use capital letters for start line